assemblyline/odm/models/submission.py

```diff
@@ -44,46 +49,47 @@ class ServiceSelection(odm.Model):
-    ignore_size = odm.Boolean(default=False, description="Ignore the file size limits?")
-    never_drop = odm.Boolean(default=False, description="Exempt from being dropped by ingester?")
-    malicious = odm.Boolean(default=False, description="Is the file submitted already known to be malicious?")
-    max_extracted = odm.Integer(default=500, description="Max number of extracted files")
-    max_supplementary = odm.Integer(default=500, description="Max number of supplementary files")
-    priority = odm.Integer(default=1000, description="Priority of the scan")
-    profile = odm.Boolean(default=False, description="Should the submission do extra profiling?")
-    psid = odm.Optional(odm.UUID(), description="Parent submission ID")
-    quota_item = odm.Boolean(default=False, description="Does this submission count against quota?")
-    services = odm.Compound(ServiceSelection, default={}, description="Service selection")
+        default=False, description="Disables dynamic recursion prevention for this submission.")
+
+    ignore_filtering = odm.Boolean(default=False, description="**TODO**: **Original**:Should we ignore filtering services? **Generated**:Indicates if filtering services should be skipped, allowing all files to be processed.")
+    ignore_size = odm.Boolean(default=False, description="Allows the submission to bypass any file size restrictions set by the system.")
+    never_drop = odm.Boolean(default=False, description="**TODO**: **Original**:Exempt from being dropped by ingester? **Generated**:Ensures the submission will not be dropped by the ingestion service, regardless of system load.")
+    malicious = odm.Boolean(default=False, description="**TODO**: **Original**:Is the file submitted already known to be malicious? **Generated**:Flags the submission as known to be malicious, possibly altering its handling.")
+    max_extracted = odm.Integer(default=500, description="The maximum number of files that can be extracted from the submission for separate analysis.")
+    max_supplementary = odm.Integer(default=500, description="**TODO**: **Original**:Max number of supplementary files **Generated**:The maximum number of supplementary files that can be generated from the submission.")
+    priority = odm.Integer(default=1000, description="**TODO**: **Original**:Priority of the scan  **Generated**:The processing priority of the submission, with lower numbers indicating higher priority.")
+    profile = odm.Boolean(default=False, description="**TODO**: **Original**:Should the submission do extra profiling?  **Generated**:Triggers additional profiling of the submission for performance and analysis metrics.")
+    psid = odm.Optional(odm.UUID(), description="The ID of a parent submission, if this submission is related to or derived from another.")
+    quota_item = odm.Boolean(default=False, description="Indicates if this submission should count against the submitter's quota.")
```

Comment:
Should this be flagged as internal use, and should always be overwritten by the API, so no use to set it by the user?

Reply:
It should but for now, we can leave it as-is.